WalletWasabi.Fluent/Models/WebcamQrReader.cs

```diff
@@ -64,13 +69,22 @@ public override async Task StartAsync(CancellationToken cancellationToken)
 
 	public override async Task StopAsync(CancellationToken cancellationToken)
 	{
+		if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+		{
+			throw new InvalidOperationException("OS not supported.");
+		}
 		Camera?.Release();
 
 		await base.StopAsync(cancellationToken);
 	}
 
 	protected override Task ActionAsync(CancellationToken cancel)
 	{
+		if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
@Szpoti  This check was necessary only in here, was it necessary in the other two methods?

And why was `[SupportedOSPlatform("windows")]` not used like the other two classes?

Reply:
What the `[SupportedOSPlatform("windows")]` attribute does is that it "pushed outwards" the warnings.
So the warnings were fixed in the source file, but was moved to the class which called the source file.
So we "pushed" out the warnings to `WebcamQrReader.cs`, and put those if statements to get rid of these warnings everywhere.

Reply:
But why we didn't use `[SupportedOSPlatform("windows")]` in `WebcamQrReader.cs` without the checks, that would fix the warnings also. Am I missing something?

Reply:
Because then the CA1416 warning would just pop up at the places where the `WebcamQrReader` is called.